cmd/render.go

```diff
@@ -3,15 +3,48 @@ package cmd
 import (
 	"fmt"
 	"io/ioutil"
+	"strings"
 
 	"github.com/resumic/schema/render"
 	"github.com/resumic/schema/theme"
 	"github.com/spf13/cobra"
 )
 
+// Added in case we change default filenames.
+const (
+	defaultHTML   = "resume.html"
+	defaultResume = "resume.json"
+)
+
 func renderRun(cmd *cobra.Command, args []string) error {
```

Comment:
Definitely a great idea and workaround, but moving to flags might be the better idea here. Sorry for not catching that sooner.
Thoughts @ArmanMazdaee @erbesharat ?

Reply:
I brilliant solution but I believe using flags would make the code simpler and also solve ambiguity on the users' side.

Reply:
Thanks guys! Definitely support the flags idea. What kind of flags are we going to want to implement?

Reply:
Commented above on that.